### PR TITLE
docs: Outline for adding checks

### PR DESCRIPTION
**Pull request by rhiaro**
Document Additional Checks in more detail to make it easier for people to add new checks.

[Preview](https://ocds-data-review-tool.readthedocs.io/en/docs-howto/how-to-add-a-validation-check.html)

### CODE REVIEW

**Comment by rhiaro** [comment]
[Write overview documentation for the DRT](https://trello.com/c/Y07UW30f/15-write-overview-documentation-for-the-drt)

**Comment by odscjames** [comment]
Reads good to me (I haven't tried to compile the RST, see comment?).

I think maybe I'd add the reason for something, as I think having that helps people understand why something is the way it is, so something like:

> 
> There are several reasons ``lib-cove-ocds`` produces results in the form of JSON dictionaries with useful keys and ``cove-ocds`` turns these into a nice display:
> *  ``cove-ocds``  is not the only tool that consumes the output of ``lib-cove-ocds``. So that each tool can produce the best UI for it's particular context, any UI work is done in the tool and not in the library
> * Doing UI work in the tool means that those UI's can be  I18N'd as each tool needs to be
> * Having machine readable output from the library means that we can take a database of output and easily parse it to create statistics, etc

Not sure if this should be elsewhere, or at any rate it's an addition so happy to approve and let you handle how you see fit.

File: docs/how-to-add-a-validation-check.rst [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
 =============================
 
+Data validation takes place in the `lib-cove-ocds <https://github.com/open-contracting/lib-cove-ocds>`_ library and are presented in the UI via templates in cove-ocds.
+
+'Common checks' process the various different objects in a release, so the data can be validated against the schema and statistics about different aspects of the data (eg. the number of contracts, the range of dates, or number of unique organisations present) can be presented in the UI. This happens in ``lib/common_checks.py``.
+
+'Additional checks' are for data validation beyond what is covered by validation against the `OCDS Schema <https://github.com/open-contracting/lib-cove-ocds/blob/master/libcoveocds/schema.py>`_ - that is, the results of these checks may suggest an issue with the data where the issue does not cause data to be invalid against the schema - and are implemented in ``lib/additional_checks.py``. An example of such a check is identifying when fields, objects and arrays exist but are empty or contain only whitespace.
+
+What follows is an outline of how to add a check. This will involve making changes in both `lib-cove-ocds` and `cove-ocds`.
```
Comment (odscjames) [inline docs/how-to-add-a-validation-check.rst:10]: Is it meant to be double ticks not single ticks? I don't have time to compile and check before a meeting tho :-)

Comment (rhiaro) [reply docs/how-to-add-a-validation-check.rst:10]: yeah it works, I already compiled: https://ocds-data-review-tool.readthedocs.io/en/docs-howto/how-to-add-a-validation-check.html

Single ticks just does italic afaik

Comment (odscjames) [reply docs/how-to-add-a-validation-check.rst:10]: Fair enough, I was confused by use of single and double ticks but now I check it looks good

**Pull request merged**